clang/test/Driver/sycl-huge-device-code-linker-script.cpp

```diff
@@ -3,10 +3,15 @@
 
 // RUN: %clangxx -### -fsycl -fsycl-link-huge-device-code %s 2>&1 | \
 // RUN:       FileCheck --check-prefix=CHECK-LINKER-SCRIPT %s
+// RUN: %clangxx -### -fopenmp -fopenmp-targets=x86_64 -flink-huge-device-code %s 2>&1 | \
+// RUN:       FileCheck --check-prefix=CHECK-LINKER-SCRIPT %s
 // CHECK-LINKER-SCRIPT: "-T" "{{.*}}.ld"
 
 // Also check that a user-provided linker script may be used:
 // RUN: %clangxx -### -fsycl -fsycl-link-huge-device-code %s \
 // RUN:       -T custom-user-script.ld 2>&1 | \
 // RUN:       FileCheck --check-prefixes=CHECK-USER-SCRIPT %s
+// RUN: %clangxx -### -fopenmp -fopenmp-targets=x86_64 %s \
+// RUN:       -T custom-user-script.ld 2>&1 | \
+// RUN:       FileCheck --check-prefixes=CHECK-USER-SCRIPT %s
```

Comment:
Test name is specific to SYCL behaviors - we could either create a new test for OpenMP behaviors or rename this test to something more fitting.

Reply:
Thanks. I decided to rename this test after the new option, with a comment describing that the old option name which is also being tested is deprecated.